### PR TITLE
[Bulk] Validation: fixaspnetdocscc1

### PR DESCRIPTION
**Pull request by ccard587**
This PR fixes about 20 aspnetdocs alternate text.

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/AspNetDocs?pullRequest=652) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/AspNetDocs?pullRequest=652) before we can accept your contribution.<br/><br/>:x: ccard587 [sign now](https://cla.dotnetfoundation.org/dotnet/AspNetDocs?pullRequest=652)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/AspNetDocs?pullRequest=652) it.</sub>

File: aspnet/whitepapers/request-validation.md [hunk 1]
```diff
@@ -31,15 +31,15 @@ The ASP.NET request validation feature proactively prevents these attacks by not
...
 
 Running this code results in a simple page that allows you to enter some text in the textbox, click the button, and display the text in the label control:
 
-![](request-validation/_static/image2.png)
+![Screenshot that shows running this code results in a simple page that allows you to enter some text in the textbox, click the button, and display the text in the label control.](request-validation/_static/image2.png)
 
 However, were JavaScript, such as `<script>alert("hello!")</script>` to be entered and submitted we would get an exception:
 
-![](request-validation/_static/image3.png)
+![If JavaScript, such as <script>alert("hello!")</script>, were entered and submitted, one would get an exception.](request-validation/_static/image3.png)
```
Comment (reemsabaw1) [inline aspnet/whitepapers/request-validation.md:42]: I suggest starting this with "Screenshot shows..."

File: aspnet/whitepapers/side-by-side-with-10.md [hunk 2]
```diff
@@ -75,17 +75,17 @@ The Aspnet\_regiis.exe provides two options for script mapping a Web application
 
 The path defines the Web application IIS metadata path, which is defined in the form of W3SVC/ROOT/{WebSiteNumber}/{Application\_Name}. For example, for a Web application called Portal located under the default Web site, the metabase path is W3SVC/1/ROOT/Portal.
 
-![](side-by-side-with-10/_static/image4.gif)
+![Screenshot that shows a Web application called Portal located under the default Web site, and the metabase path is W3SVC/1/ROOT/Portal.](side-by-side-with-10/_static/image4.gif)
```
Comment (reemsabaw1) [inline aspnet/whitepapers/side-by-side-with-10.md:78]: "W3SVC/1/ROOT/Portal" might be difficult for a screen-reader to read out. Try writing out this part of the text, for example: W 3 S V C slash 1 slash ROOT slash Portal.

File: aspnet/whitepapers/side-by-side-with-10.md [hunk 3]
```diff
@@ -75,17 +75,17 @@ The Aspnet\_regiis.exe provides two options for script mapping a Web application
...
 
 Note You can also use a tool called the Metabase Editor to get the metabase path. You can download this tool from the Microsoft Support site at [https://support.microsoft.com/default.aspx?scid=kb;en-us;232068.](https://support.microsoft.com/default.aspx?scid=kb;en-us;232068)
 
 - Run Aspnet\_regiis.exe -s W3SVC/1/ROOT/Portal to update the portal IIS script map and its subapplication.  
   
-    ![](side-by-side-with-10/_static/image5.gif)
+    ![Screenshot that shows to run Aspnet\_regiis.exe -s W3SVC/1/ROOT/Portal to update the portal IIS script map and its subapplication.](side-by-side-with-10/_static/image5.gif)
```
Comment (reemsabaw1) [inline aspnet/whitepapers/side-by-side-with-10.md:84]: try writing out W3SVC/1/ROOT/Portal so it is easier for a screen reader to read out loud.

File: aspnet/whitepapers/side-by-side-with-10.md [hunk 4]
```diff
@@ -75,17 +75,17 @@ The Aspnet\_regiis.exe provides two options for script mapping a Web application
...
 
 - Run Aspnet\_regiis.exe -sn W3SVC/1/ROOT/Portal to update the portal IIS script map, without affecting applications in the portal?s subdirectories.  
   
-    ![](side-by-side-with-10/_static/image6.gif)
+    ![Screenshot that shows to Run Aspnet\_regiis.exe -sn W3SVC/1/ROOT/Portal to update the portal IIS script map, without affecting applications in the portal?s subdirectories.](side-by-side-with-10/_static/image6.gif)
```
Comment (reemsabaw1) [inline aspnet/whitepapers/side-by-side-with-10.md:88]: try writing out W3SVC/1/ROOT/Portal so it is easier for a screen reader to read out loud.

Also, portal?s is a typo.

File: aspnet/whitepapers/side-by-side-with-10.md [hunk 5]
```diff
@@ -99,19 +99,19 @@ An administrator can use the Internet Service Manager to find which version of t
...
-    ![](side-by-side-with-10/_static/image9.gif)
+    ![Screenshot that shows to right-click on the Web application, and click on Properties.](side-by-side-with-10/_static/image9.gif)
 6. From the Property window, select **Configuration.**  
   
-    ![](side-by-side-with-10/_static/image10.gif)
+    ![Screenshot that shows from the Property window, select Configuration.](side-by-side-with-10/_static/image10.gif)
 7. From the application mapping table, select **.aspx**, and click **Edit**.  
   
-    ![](side-by-side-with-10/_static/image11.gif)
+    ![Screenshot that shows from the application mapping table, select .aspx, and click Edit.](side-by-side-with-10/_static/image11.gif)
```
Comment (reemsabaw1) [inline aspnet/whitepapers/side-by-side-with-10.md:114]: try spacing out .aspx

File: aspnet/whitepapers/side-by-side-with-10.md [hunk 6]
```diff
@@ -99,19 +99,19 @@ An administrator can use the Internet Service Manager to find which version of t
...
 8. From the **Executable** text box, look at the version directory by scrolling. If the version directory is v.1.1.4322, the application is mapped to .NET Framework 1.1. Conversely, if the version directory is v1.0.3705, the application is mapped to .NET Framework 1.0.  
   
-    ![](side-by-side-with-10/_static/image12.gif)
+    ![Screenshot that shows the Executable text box and the version directory v.1.1.4322 from scrolling, which is mapped to a .NET Framework 1.1.](side-by-side-with-10/_static/image12.gif)
```
Comment (reemsabaw1) [inline aspnet/whitepapers/side-by-side-with-10.md:117]: try spacing out v.1.1.4322.

**Comment by ccard587** [comment]
Global effort to fix build validation errors

@Rick-Anderson  - MSFT NOTE: The validation warnings and suggestions in this PR will be fixed by another author in another PR.

The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you'd like to see. Thank you!

**Review by Rick-Anderson — APPROVED** [review]

**Pull request merged**